launch_chffrplus.sh

```diff
@@ -1,20 +1,9 @@
 #!/usr/bin/bash
 
-export OMP_NUM_THREADS=1
-export MKL_NUM_THREADS=1
-export NUMEXPR_NUM_THREADS=1
-export OPENBLAS_NUM_THREADS=1
-export VECLIB_MAXIMUM_THREADS=1
-
 if [ -z "$BASEDIR" ]; then
   BASEDIR="/data/openpilot"
 fi
-
-if [ -z "$PASSIVE" ]; then
-  export PASSIVE="1"
-fi
-
-STAGING_ROOT="/data/safe_staging"
+source "$BASEDIR/launch_env.sh"
```

Comment:
What's the point of breaking up the launch script? It seems like nothing else uses `launch_env.sh`

Reply:
Don't want to track the openpilot required NEOS version in two places, can't source `launch_chffrplus.sh` as it exists. Chatted with Willem about maybe moving OP version to a bare standalone file like the EON `/VERSION` but we settled on this.